src/Core/Application/Platform/Service/PlatformVersionServiceInterface.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * Copyright 2005 - 2021 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+declare(strict_types=1);
+
+namespace Core\Application\Platform\Service;
+
+interface PlatformVersionServiceInterface
+{
+    /**
+     * Get the upgrade version if an upgrade is available.
+     *
+     * @return string|null
+     */
+    public function getWebUpgradeVersion(): ?string;
```

Comment:
later, we'll may need to create a value object `Version`

Reply:
yes, I thought about it